Removed unnecessary parameters in constructor

Chiearf-Chaotic opened the pull request:
- jda field can be set through guild, member field can be set through message.

Chiearf-Chaotic committed:
Removed unnecessary parameters in constructor

- jda field can be set through guild, member field can be set through message.

Blayyke reviewed (changes requested):
If you wanna do that, guild can be set through message too

Chiearf-Chaotic commented:
Yes, did forgot that message also have getters for guild.

Chiearf-Chaotic committed:
Blayyke - guild field can be set through messages.

- Made changes to the fields according to blayyke.
- Also made jda field set through message.
